podkrepi-bg/frontend
# Add manual import of failed bank donation imports

Adds the functionality to manually add the correct campaign payment code for bank donations which were not recognized during automated importing

It depend on this API PR:  
podkrepi-bg/api#488

## Motivation and context







## Screenshots:



Before|After
---|
![image](https://user-images.githubusercontent.com/78339364/233779102-2d7701e7-73af-4492-b441-f804d06bd7a6.png)
![image](https://user-images.githubusercontent.com/78339364/233779142-c3cc15e9-66cb-41ae-ad3d-983f87e856f6.png)
![image](https://user-images.githubusercontent.com/78339364/233779160-3229f528-9caa-4e32-af9a-f663f5135d72.png)

Paste screenshot|Paste screenshot



## Testing

### Steps to test
 Go to the bank transactions page and assign the correct campaign code for the unrecognized transaction. GO check that the campaign vault has been incremented with that amount
